packages/webapp/src/plugins/cache.js

```diff
@@ -1,17 +1,22 @@
-import { Engine as CatboxRedis } from '@hapi/catbox-redis'
-import { REDIS_HOST, REDIS_PORT, REDIS_PASSWORD, REDIS_TLS } from '../utils/config.js'
+import { Engine as CatboxMemory } from '@hapi/catbox-memory'
+import { CACHE_MAX_BYTE_SIZE, CACHE_CLEANUP_INT_MIN_SEC, CACHE_CLONE_BUFFER_ON } from '../utils/config.js'
+
+const options = {}
+if (CACHE_MAX_BYTE_SIZE) {
+  options.maxByteSize = CACHE_MAX_BYTE_SIZE
+}
+if (CACHE_CLEANUP_INT_MIN_SEC) {
+  options.minCleanupIntervalMsec = CACHE_CLEANUP_INT_MIN_SEC
+}
+if (CACHE_CLONE_BUFFER_ON) {
+  options.cloneBuffersOnGet = CACHE_CLONE_BUFFER_ON
+}
```

Comment:
I am interested - could you please send me a docs link on what these values should be in local, prod?

Reply:
Sure (they were already in the jira as well): https://hapi.dev/module/catbox-memory/api/?v=6.0.1#new-catboxmemoryengineoptions.

You don't need to set these if you are happy with the defaults.